src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2460,4 +2461,22 @@ export default class AircraftModel {
     removeConflict(conflictingAircraft) {
         delete this.conflicts[conflictingAircraft.callsign];
     }
+
+    /**
+     * @for AircraftModel
+     * @method isValidSpeed
+     * @type {boolean}
+     */
```

Comment:
These `isValid` methods look good except they are handled n the wrong class. My mistake, I may not have explained that very well before. 

These two methods should be moved to the `AircraftTypeDefnitionModel` and references should be updated.